docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-web-data-model/conversions/index.md

```diff
@@ -0,0 +1,204 @@
+      formData={formData}
+      onChange={(e) => setFormData(e.formData)}
+      validator={validator}
+      showErrorList='bottom'
+      liveValidate
+    ><div/></Form>
+    <h4>Project Variable:</h4>
+    <CodeBlock language="yaml">snowplow__conversion_events: {JSON.stringify(formData, null, 4)}</CodeBlock>
+    </>
+  );
+};
+```
+
+<Badges badgeType="dbt-package Release" pkg="web"></Badges>
+
+# Conversions
+
+Conversion events are a type of event that's important to your business, be that a transaction, a sign up to a newsletter, or the view of a specific page. Whatever type of event matters to you, so long as it can be determined from a single event record, you'll be able to model and aggregate these conversions at a session level with our package.
+
+Because we know that each user may have a different concept of what a conversion means to them, for example it could be a specific type of event such as a `page_view` on a specific url, a self-describing event such as `sign_up`, or even a bespoke `conversion` event type with some attached context, the way we model conversions is also flexible and allows you to change your definition over time. This allows you to track multiple types of conversions, with varying logic, even retrospectively.
+
+:::caution
+
+Because this is part of the sessions table within the web package, we still expect your sessions to contain at least one `page_view` or `page_ping` event, and the events must all have a `domain_sessionid` to be included in the `base_events_this_run_table`. Without a `domain_sessionid` the event will not be visible to the model, and without a `page_view` or `page_ping` in the session there will be no session record for the model to attach the conversions to.
+
+:::
+
+## Conversion Columns
+For every type of conversion you provide a configuration for, the package will add up to 6 columns to the `snowplow_web_sessions` table, depending on what you provide in your configuration. All columns start with `cv_{name}_` based on the name of your conversion and will be placed at the end of the table.
+
+| Column                       | Description                                                                                  |
+| ---------------------------- | -------------------------------------------------------------------------------------------- |
+| `cv_{name}_volume`           | An integer column with the total number of conversions in the session                        |
+| `cv_{name}_values`           | An array of values for each conversion, if provided in the configuration                     |
+| `cv_{name}_total`            | The total of the values for each conversion, if provided in the configuration                |
+| `cv_{name}_events`           | An array of `event_id`s for each conversion, if `list_events` is `true` in the configuration |
+| `cv_{name}_first_conversion` | The `collector_tstamp` of the first conversion event                                         |
```

Comment:
We should probably change this to `derived_tstamp` so that we can allow for historical conversions to be sent in.